.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python3.7"
+}
```

Comment:
@Genza999 please donot push .vscode/settings. Add to gitignore

Reply:
Thanks. Done!

Reply:
LGTM